roles/kubernetes/node/tasks/kubelet.yml

```diff
@@ -63,9 +63,10 @@
   meta: flush_handlers
 
 - name: Enable kubelet
-  service:
+  systemd:
     name: kubelet
     enabled: yes
     state: started
+    daemon_reload: yes
```

Comment:
"restart kubelet" handler (containing deamon-reload and kubelet restart) is triggered in the task before.
Thus there is no need to add daemon_reload.